Packet++/header/GvcpLayer.h

```diff
@@ -0,0 +1,758 @@
+#pragma once
+
+#include "IpAddress.h"
+#include "Layer.h"
+#include "MacAddress.h"
+#include <iostream>
+
+/// @file
+
+/// This GVCP implementation is based on GigE Vision ® Specification version 2.0
+
+/**
+ * @namespace pcpp
+ * @brief The main namespace for the PcapPlusPlus lib
+ */
+namespace pcpp
+{
+	uint16_t hostToNet16(uint16_t val);  // forward declare the function in SystemUtil.h
+	uint16_t netToHost16(uint16_t val);  // forward declare the function in SystemUtil.h
+
+	namespace internal
+	{
+		static constexpr size_t kGvcpMagicNumber = 0x42;
+		static constexpr size_t kGvcpRequestHeaderLength = 8;
+		static constexpr size_t kGvcpAckHeaderLength = 8;
+		static constexpr size_t kGvcpDiscoveryBodyLength = 248;
+		static constexpr size_t kGvcpForceIpBodyLength = 56;
+	}  // namespace internal
+
+	using GvcpFlag = uint8_t;  // flag bits are specified by each command
+
+	/// @brief Gvcp command defines the command values and the corresponding acknowledge values
```

Comment:
1. nit: let's use GVCP (capital letters) instead of Gvcp in the doxygen documentation (not in class names etc).
Please search and replace other instances of Gvcp in this file and replace them too
2. In order to keep the convention we have everywhere else, maybe use this format for doxygen comments?
    ```cpp
    /**
     * comment...
     */
    ```

Reply:
sure fixed.